docs/product-development-guide/src/main/asciidoc/security-management-sec-provider-con.adoc

```diff
@@ -0,0 +1,10 @@
+[id='security-management-sec-provider.adoc']
+= Security Provider
+
+A security environment is usually provided by the use of a realm. Realms are used to restrict access to the different application’s resources. Realms contains information about the users, groups, roles, permissions and any other related information.
+
+Typically, the application’s security is delegated to the container’s security mechanism, which consumes a particular realm at same time. It is important to consider that  several realm implementations exist. For example Wildfly provides a realm based on the `application-users.properties/application-roles.properties` files, Tomcat provides a realm based on the `tomcat-users.xml` file, and so on.  The security realm can be different for each installation.
```

Comment:
Instead of using application's security, can we say 'application security' or 'security of the application' to avoid possessive forms. Same comment for "Container's".